src/components/ui/Icon/README.md

```diff
@@ -0,0 +1,167 @@
+
+### Don't use an `Icon` for...
+
+- Displaying a complex image or photo. Use an `Image` instead.
+- Communicating a link, button, or action. Use a `Button` or `Tab` (combined with an `Icon`) instead.
+
+## Appearance
+
+### Colour
+
+An icon can be passed either a stroke colour or a fill colour.
+
+`strokeColor` affects the outline of the icon:
+
+```jsx
+<Icon name="pen-tool" strokeColor="#053e6c" large />
+<Icon name="target" strokeColor="#8a0000" large />
+<Icon name="star" strokeColor="#eeb600" large />
+```
+
+`fillColor` affects the inside of the icon:
+
+```jsx
+<Icon name="pen-tool" fillColor="#053e6c" large />
+<Icon name="target" fillColor="#8a0000" large />
+<Icon name="star" fillColor="#eeb600" large />
+```
+
+You can combine both colours if you like:
+
+```jsx
+<Icon name="pen-tool" fillColor="#5788b1" strokeColor="#043258" large />
+<Icon name="target" fillColor="#c26767" strokeColor="#780000" large />
+<Icon name="star" fillColor="#f7e095" strokeColor="#d7a400" large />
+```
+
+When pairing an icon with a label, use a slightly lighter variant of the text colour for the icon colour. This make the two elements feel connected, whilst ensuring the icon doesn't have more visual weight than the text. (Nautilus does this by default because we always want you to look good. 🙌)
+
+```jsx
+import { Paragraph } from '@octopusthink/nautilus';
+
+<React.Fragment>
+  <Paragraph style={{ color: '#b32973' }}>
+    <Icon name="heart"/> I love Lucy.
+  </Paragraph>
+  <Paragraph>
+    <Icon name="coffee" />
+    Lucy loves coffee.
+  </Paragraph>
+</React.Fragment>
+```
+
+### Size
+
+Icons are available in five different sizes: `extraLarge` (`64px`), `large` (`48px`), `medium` (`24px`), `small` (`16px`), and `extraSmall` (`8px`). If you don't pass a size value, your icons will use the `medium` size.
+
+The default icon sizes can be tweaked in your theme. Sizes should be defined in `rem` to ensure the icons adjust to the user's font size. (The above sizes are the pixel equivalents of the `rem` sizes provided for reference.)
+
+```jsx
+<Icon name="heart" extraSmall />
+<Icon name="heart" small />
+<Icon name="heart" medium />
+<Icon name="heart" large />
+<Icon name="heart" extraLarge />
+
+<Icon name="star" extraSmall />
+<Icon name="star" small />
+<Icon name="star" medium />
+<Icon name="star" large />
+<Icon name="star" extraLarge />
+
+<Icon name="pen-tool" extraSmall />
+<Icon name="pen-tool" small />
+<Icon name="pen-tool" medium />
+<Icon name="pen-tool" large />
+<Icon name="pen-tool" extraLarge />
+```
+
+Stroke widths scale with the icon size for better legibility. Stroke widths are also customisable via your theme.
+
+
+### Margin & alignment
+
+By default, an `Icon` has a small amount of margin to its right, to give it some breathing space when used with text.
```

Comment:
> small amount of margin to its right

I wonder if we should write "following the Icon" rather than "to its right" for RTL users. 🤷‍♂

Reply:
We sure should! I've tweaked this too.